src/exercise/05.js

```diff
@@ -4,24 +4,16 @@
 import * as React from 'react'
 import '../box-styles.css'
 
-// 💰 Use the className for the size and style (backgroundColor) for the color
-// 💰 each of the elements should also have the "box" className applied
-
-// 🐨 add a className prop to each of these and apply the correct class names
-// 💰 Here are the available class names: box, box--large, box--medium, box--small
-
-// 🐨 add a style prop to each of them as well so their background color
-// matches what the text says it should be as well as `fontStyle: 'italic'`
-const smallBox = <div>small lightblue box</div>
-const mediumBox = <div>medium pink box</div>
-const largeBox = <div>large orange box</div>
+function Box(props) {
+  return <div className={"box box--" + props.boxSize} style={{backgroundColor: props.boxColor}}>{props.boxText}</div>
```

Comment:
Curious why concatenate and not `string template ${literal}`?

Reply:
No good reason, I'll make it a string literal, its easier to read.